### ISSUE TITLE
Incipit images

### ISSUE THREAD

**Issue by juliacmcf** [diamm/diamm#265]
A number of sources have multiple incipit image files (e.g. the original incipit from RISM which redirects to another; the one it redirects to; a new one created by Thomas Schmidt). I have not checked to see if all three have come through on the migration, but from the editing interface for items it looks as if it's only possible to have one.

**Comment by ahankinson** [comment]
Yes, it is currently only possible to have one incipit. It would be a pretty big change to allow more than one too. 

How urgent? We're not displaying the incipit at the moment. Should we?

**Comment by juliacmcf** [comment]
It is important for sources where we don’t have images, but not particularly otherwise (for obvious reasons!).
